app/assets/stylesheets/application.scss

```diff
@@ -33,9 +33,9 @@
 }
 
 .dashed {
-  border: 2px dashed #808080;
-  height: 93%;
-  width: 93%;
+   border: 2px dashed #D3D3D3;
```

Comment:
Color literals like `#D3D3D3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D3D3D3` should be written as `#d3d3d3`<br>Line should be indented 2 spaces, but was indented 3 spaces